### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by yakschuss**
Brightline uses ViewComponents!

### Summary

This updates `index.md` with [Brightline](https://www.hellobrightline.com) as a user of ViewComponent!

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -202,6 +202,7 @@ ViewComponent is built by over a hundred members of the community, including:
 * [Shogun](https://getshogun.com/)
 * [Cometeer](https://cometeer.com/)
 * [Framework](https://frame.work/)
+* [Brightline](https://hellobrightline.com)
```
Comment (Spone) [inline docs/index.md:205]: Can you reorder the list alphabetically please?

We [talked about it](https://github.com/github/view_component/pull/1287#issuecomment-1049451765) previously but did not do it yet.

Comment (yakschuss) [reply docs/index.md:205]: Sure thing!

Comment (yakschuss) [reply docs/index.md]: @Spone I'm not sure my CHANGELOG change was in the right place. Can you confirm?

**Comment by hjhart** [comment]
I work at Brightline and can confirm that we use ViewComponent. :)

File: docs/index.md (hunk 1 above)
Comment (Spone) [reply docs/index.md]: Please move it to the top, just below the `main` heading :)

**Pull request merged**